src/core/linalg/src/sparse/4C_linalg_vector.hpp

```diff
@@ -407,50 +411,12 @@ namespace Core::LinAlg
     std::shared_ptr<Epetra_IntVector> vector_;
   };
 
-  /**
-   * Temporary helper class for migration. View one of the Trilinos vector types as one of ours.
-   * Make sure that the viewed vector lives longer than the view.
-   */
-  template <typename VectorType>
-  class VectorView
-  {
-   public:
-    template <typename EpetraVectorType>
-    VectorView(EpetraVectorType& vector) : view_vector_(VectorType::create_view(vector))
-    {
-    }
-
-    // Make the class hard to misuse and disallow copy and move.
-    VectorView(const VectorView& other) = delete;
-    VectorView& operator=(const VectorView& other) = delete;
-    VectorView(VectorView&& other) = delete;
-    VectorView& operator=(VectorView&& other) = delete;
-    ~VectorView() = default;
 
-    //! Allow implicit conversion to Vector for use in new interfaces.
-    operator VectorType&() { return *view_vector_; }
-
-    //! Allow implicit conversion to RCP<Vector> for use in new interfaces.
-    std::shared_ptr<VectorType>& get_non_owning_rcp_ref() { return view_vector_; }
-
-   private:
-    //! Store inside an RCP because our interfaces frequently take references to RCPs.
-    std::shared_ptr<VectorType> view_vector_;
+  template <>
+  struct WrapperFor<Epetra_Vector>
+  {
+    using type = Vector<double>;
   };
```

Comment:
Here is the specialization. Together with the static method, this is enough for Views to work :)